src/eduid_webapp/security/views/webauthn.py

```diff
@@ -76,9 +74,11 @@ def registration_begin(user):
     current_app.logger.debug('Webauthn Registration data: {}.'.format(registration_data))
     current_app.stats.count(name='webauthn_register_begin')
 
-    cbor_data = cbor.dumps(registration_data)
-    current_app.logger.debug('CBOR encoded Registration data: {}.'.format(cbor_data))
-    return Response(response=cbor_data, status=200, mimetype='application/cbor')
+    encoded_data = base64.b64encode(cbor.dumps(registration_data)).decode('ascii')
```

Comment:
Should not decode('utf-8') work as json by default is utf-8?

Reply:
But what we are decoding is base64 bytes, which is ascii, I just provide it as unicode so json.dumps produces utf-8 out of it